src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -25,17 +25,18 @@
 					tw-w-full
 					tw-bg-black
 					tw-rounded
+					tw-object-contain
 				"
 					:alt="'photo of ' + borrowerName"
 					:aspect-ratio="3 / 4"
 					:default-image="{ width: 336 }"
 					:hash="imageHash"
 					:images="[
-						{ width: 336, viewSize: 1024 },
-						{ width: 336, viewSize: 768 },
-						{ width: 416, viewSize: 480 },
-						{ width: 374, viewSize: 414 },
-						{ width: 335, viewSize: 375 },
+						{ width: 336, faceZoom: 0.7, viewSize: 1024 },
```

Comment:
`faceZoom` can just be `1` as it's converted in code to a whole number.

Otherwise this is a really good improvement.